static/app/components/onboardingWizard/filterSupportedTasks.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  feedbackOnboardingPlatforms,
+  replayOnboardingPlatforms,
+  withoutPerformanceSupport,
+} from 'sentry/data/platformCategories';
+import {type OnboardingTask, OnboardingTaskKey, type Project} from 'sentry/types';
+
+const replayRelatedTasks = [OnboardingTaskKey.SESSION_REPLAY];
+const performanceRelatedTasks = [
+  OnboardingTaskKey.FIRST_TRANSACTION,
+  OnboardingTaskKey.PERFORMANCE_GUIDE,
+  OnboardingTaskKey.METRIC_ALERT,
+];
+const feedbackRelatedTasks = [OnboardingTaskKey.USER_REPORTS];
+
+export function filterSupportedTasks(
+  projects: Project[] | undefined,
+  allTasks: OnboardingTask[]
+): OnboardingTask[] {
+  if (!projects) {
+    return [];
+  }
+  // Remove tasks for features that are not supported
+  return allTasks.filter(
+    task =>
+      (shouldShowReplayTasks(projects) || !replayRelatedTasks.includes(task.task)) &&
```

Comment:
oh man, this is a little hard to read for me :(
Any chance we could do this different in a loop with if statements?

Reply:
I made it a little better by filter out instead of filtering in. I don't think a for loop is the right fit here though